### PR TITLE
Fix: status label background color naming

### PR DESCRIPTION
**Pull request by yo1995**
There was a typo in the color asset folder for `statusLabelBackgroundColor`.

This PR...

- Renames the color asset
- Fix all occurrences of typo `statusLabelBackgroudColor` in the storyboards
- Remove the class variable for `UIColor` that is unused in any sample, which was first discussed here: https://github.com/Esri/arcgis-runtime-samples-ios/pull/899#issuecomment-902067580

### CODE REVIEW

File: arcgis-ios-sdk-samples/Display information/Show callout/Callout.storyboard [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="17156" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="YcZ-XH-Sb3">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="18122" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="YcZ-XH-Sb3">
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Display information/Show callout/Callout.storyboard:2]: Ah I wasn't mean to make these changes. 🤦  It was introduced when I verify the changes by opening the storyboard.
```suggestion
<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="17156" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="YcZ-XH-Sb3">
```

File: arcgis-ios-sdk-samples/Display information/Show callout/Callout.storyboard [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
     <device id="retina6_1" orientation="portrait" appearance="light"/>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="17126"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="18093"/>
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Display information/Show callout/Callout.storyboard:6]:
```suggestion
        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="17126"/>
```

**Review by philium — APPROVED** [review]

**Review by saratchandrakarumuri — APPROVED** [review]
Randomly verified bunch of effected samples and they all look good with dark mode

**Comment by yo1995** [comment]
Thank you!

**Pull request merged**